MatrixKit/Controllers/MXKAuthenticationViewController.m

```diff
@@ -89,6 +89,39 @@ + (instancetype)authenticationViewController
                                           bundle:[NSBundle bundleForClass:[MXKAuthenticationViewController class]]];
 }
 
+#pragma mark -
+
+- (instancetype)initWithNibName:(nullable NSString *)nibNameOrNil bundle:(nullable NSBundle *)nibBundleOrNil
```

Comment:
We should really add these methods to top parent (MXKViewController and MXKTableViewController).